test/grnstate-tests.js

```diff
@@ -1,22 +1,23 @@
-// var assert = require("chai").assert;
-// import { grnState } from "./grnstate";
-
-/* TODO: Framework for front-end test
-
-var testResetNormalization = (input, propertyExpected) => {
-    assert.equal(propertyExpected, grnState.propertyValue);
-}
+/* eslint-disable max-len */
+/* eslint no-unused-vars: [2, {"varsIgnorePattern": "should"}] */
+var should = require("chai").should();
+import { grnState } from "../web-client/public/js/grnstate";
 
 describe("The grnState model object", () => {
-    it("updates the reset normalization maximum correctly", () => {
+    it("should update the reset normalization maximum correctly", () => {
         grnState.network = {
-            Build a network here. Because reset normalization max only relies on positiveWeights and negativeWeights,
-            your test network may contain just those properties.
+            positiveWeights: [0.7715106466403678, 5.212343052095555, 0.826365326790911, 1.5180198066137216, 0.27964603867183396, 2.9870960868914778, 0.7743557796125339, 0.6701684007311992, 0.22069064019031245, 0.2841810246722045, 0.08852330457981578, 0.5563132195423768],
+            negativeWeights: [-1.1672013354497313, -2.4438462486461163, -1.485274408559026, -0.7371671668856121, -2.9527940988546217, -3.3875125167057103, -0.0735729607899455, -0.12373154839822605, -0.5487650216350863, -0.442904423529279, -0.8662459811465724, -2.850006860838391, -1.5733259134754158, -0.02279063304065471, -1.9832439149172112, -0.25954746744906587],
```

Comment:
Was this the reason for `disable max-len`? If so, then break lines! No one wants to read a very long array expression in a single line anyway.